ui/src/duck/designation.ts

```diff
@@ -0,0 +1,42 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+
+import { Designation, fetch } from 'src/entities/Designation';
+
+export interface DesignationState {
+  error?: string;
```

Comment:
Let's get rid of error if we are not using it. It'll only end up misleading someone that we are handling this error.